_includes/cloud/cli-get-started.md

```diff
@@ -1,35 +1,35 @@
 
-1.	Log in to your local development system, or switch to, the [Magento file system owner]({{ page.baseurl }}/cloud/before/before-workspace-file-sys-owner.html).
-2.	Change to a directory to which the Magento file system owner has write access.
-2.	Enter the following command in a terminal to log in to your project:
+1. Log in to your local development system, or switch to, the [Magento file system owner]({{ page.baseurl }}/cloud/before/before-workspace-file-sys-owner.html).
+2. Change to a directory to which the Magento file system owner has write access.
+2. Enter the following command in a terminal to log in to your project:
 
-		magento-cloud login
-3.	List your projects. With the project ID, you can complete additional commands.
+  magento-cloud login
```

Comment:
A step that includes code blocks, images, or step sub-paragraphs should begin at the same position as the first letter of the step sentence. Otherwise, this ordered list loses the order and formatting. You will have to review each file to ensure that replacing the hard tab does not adversely affect the formatting of the topic elements.